src/ert/config/ert_config.py

```diff
@@ -182,6 +187,10 @@ def from_dict(cls, config_dict) -> Self:
             installed_forward_model_steps = (
                 cls._installed_forward_model_steps_from_dict(config_dict)
             )
+
+            installed_forward_model_steps.update(
```

Comment:
This means that plugins overrwite user specified which I think should be the other way around.